manifest/v1alpha/budgetadjustment/validation.go

```diff
@@ -72,3 +74,23 @@ var sloValidationRule = govy.New(
 		Required().
 		Rules(rules.StringDNSLabel()),
 )
+
+var atLeastHourlyFreq = govy.NewRule(func(rule *rrule.RRule) error {
```

Comment:
bysecond and byminute still generate more events then we allow:
EXAMPLE:
`"FREQ=HOURLY;BYHOUR=6,8,9,10,11,12,13,14,15,16;BYMINUTE=6,8,9,10,11,12,13,14,15,16,59;COUNT=1001"`